statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -574,6 +574,7 @@ def smooth(self, params, transformed=True, complex_step=False,
 
         return result
 
+    # TODO: _handle_args does not use `self`.  Does it need to be a method?
```

Comment:
I think we should leave these TODOs out and maybe create an issue instead. I find that in older code I find in Statsmodels, the TODOs tend to be unclear and maybe don't get deleted when they get fixed.